evm-verifier/contracts/EVMProofHelper.sol

```diff
@@ -125,10 +119,7 @@ library EVMProofHelper {
             (bool isDynamic, uint256 slot) = computeFirstSlot(command, constants, values);
             if(!isDynamic) {
                 values[i] = abi.encode(getFixedValue(target, getter, storageRoot, slot, proof[proofIdx++]));
-                if(values[i].length > 32) {
```

Comment:
Why remove this check?

Reply:
bytes32 can't encode to anything other than 32 bytes